Web/Design a Library Management System/MemberService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Design_a_Library_Management_System.Model;
+using DesignLibraryManagementSystem.Models;
+using Library;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Design_a_Library_Management_System
+{
+    public class MemberService : ControllerBase, IMemberService
```

Comment:
```suggestion
    public class MemberService : IMemberService
```
why do you need ControllerBase?